kafka-connect/kafka-connect-runtime/hive/NOTICE

```diff
@@ -1399,7 +1221,7 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-Group: org.apache.orc  Name: orc-shims  Version: 1.9.4
+Group: org.apache.orc  Name: orc-shims  Version: 1.9.5
```

Comment:
Same as the Apache Kerby comment, since it's ASF we could technically just remove it from the notice? Non-blocking though since my understanding is it's not a must to not have it.